### PR TITLE
Fix ProposalCreated event dates

### PR DESCRIPTION
**Pull request by Michael-A-Heuer**
## Description

Fix for https://discord.com/channels/672466989217873929/1102145446962540565

Task: [OS-409](https://aragonassociation.atlassian.net/browse/OS-409)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[OS-409]: https://aragonassociation.atlassian.net/browse/OS-409?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/contracts/deploy/update/to_v1.3.0/11_TokenVoting_Plugin_conclude.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log('\nConcluding TokenVoting Plugin Update');
+
+  hre.aragonToVerifyContracts.push(
+    await hre.deployments.get('TokenVotingSetup')
```
Comment (Rekard0) [inline packages/contracts/deploy/update/to_v1.3.0/11_TokenVoting_Plugin_conclude.ts:7]: are we sure `TokenVotingSetup` is the new build and not the old one?
i would rather use a diferent name as mentioned in my previous comment, or write a test to be sure

**Review by brickpop — DISMISSED** [review]

File: packages/contracts/deploy/update/to_v1.3.0/10_TokenVoting_Plugin.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import tokenVotingReleaseMetadata from '../../../src/plugins/governance/majority-voting/token/release-metadata.json';
+import tokenVotingBuildMetadata from '../../../src/plugins/governance/majority-voting/token/build-metadata.json';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log('\nUpdate TokenVoting Plugin');
+  const {deployments, ethers, network} = hre;
+  const {deploy} = deployments;
+  const [deployer] = await ethers.getSigners();
+
+  const deployResult = await deploy('TokenVotingSetup', {
```
[the remark this answers is not in the document]
Comment (Michael-A-Heuer) [reply packages/contracts/deploy/update/to_v1.3.0/10_TokenVoting_Plugin.ts:15]: Correct, had this on the back of my mind but forgot to come back to it

**Review by brickpop — APPROVED** [review]

**Review by Rekard0 — APPROVED** [review]
LGTM

File: packages/contracts/deploy/update/to_v1.3.0/10_TokenVoting_Plugin.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Michael-A-Heuer) [reply packages/contracts/deploy/update/to_v1.3.0/10_TokenVoting_Plugin.ts]: Right, we will do it in 
https://aragonassociation.atlassian.net/browse/OS-416
and test it in
https://aragonassociation.atlassian.net/browse/OS-441

**Pull request merged**